cmd/expired-authz-purger/main.go

```diff
@@ -172,7 +195,11 @@ func main() {
 		fmt.Fprintln(os.Stderr, "Grace period is 0, refusing to purge all pending authorizations")
 		os.Exit(1)
 	}
+	if config.ExpiredAuthzPurger.Parallelism < 1 {
```

Comment:
In order to deal with not wanting negative values here `Parallelism` could just be a `uint`. Also this requires a config change to be able to be run, which is probably fine but technically breaks the configurability guidelines. It might be nice to just do a check for `Parallelism == 0` and set it to `1` in that case.

Reply:
That's a better suggestion than my own to check for `< 1`.

Reply:
Ping on the configuration/deployability part of this, also now you've switched to `uint` even if you don't want to manually set it to `1` the check should probably just be `Parallelism == 0`.

Reply:
Since this doesn't (yet) get run automatically, and can't reasonably run without configuring a higher parallelism, I would prefer to land it as-is with the strict check, even though that means the purger can't be run with the current config. The strict check is more valuable in the long run to ensure that a specific choice was made for parallelism.

Reply:
:+1: